.github/CODEOWNERS

```diff
@@ -1,3 +1,4 @@
 # CI
 /.github/ @paritytech/ci @pepoviola
```

Comment:
Is this group exists?

```
Unknown owner on line 2: make sure the team @paritytech/ci exists and has write access to the repository
/.github/ @paritytech/ci @pepoviola
```

Reply:
ping @sergejparity

Reply:
The group exists but should be configured in the repo settings (Settings -> Collaborators and teams -> Add teams -> @ci with write permissions). From our team only @TriplEight has access

Reply:
Great, Thanks!